command/cmd_list.go

```diff
@@ -158,12 +158,18 @@ func (l ListMessage) humanize() string {
 }
 
 const (
-	listFormat = "%19s %1s %-6s %12s %s"
 	dateFormat = "2006/01/02 15:04:05"
 )
 
 // String returns the string representation of ListMessage.
 func (l ListMessage) String() string {
+	var listFormat = "%19s %1s %-1s %12s %s"
+	var etag string
+	if l.showEtag {
```

Comment:
could you add an e2e test case?

Reply:
sure

Reply:
I added alignment() function to check the last indexes of strings. But since gofakess3 does not support tags (returns ?). It won't check if ETag flag brokes the alignment of output.  We can revert this change if you wish, or keep it for safety.